packages/sdk/src/cli/checkUnsupportedDataProxy.ts

```diff
@@ -0,0 +1,76 @@
+import chalk from 'chalk'
+import { O } from 'ts-toolbelt'
+
+import { getConfig, getSchemaPath, link } from '..'
+import { loadEnvFile } from '../utils/loadEnvFile'
+
+/**
+ * These are the cli args that we check the data proxy for. If in use
+ */
+const checkedArgs = {
+  '--url': true,
+  '--to-url': true,
+  '--from-url': true,
+  '--shadow-database-url': true,
+  '--schema': true,
+  '--from-schema-datamodel': true,
+  '--to-schema-datamodel': true,
+}
+
+type Args = O.Optional<O.Update<typeof checkedArgs, any, string>>
+
+/**
+ * Get the message to display when a command is forbidden with a data proxy flag
+ * @param command the cli command (eg. db push)
+ * @returns
+ */
+export const forbiddenCmdWithDataProxyFlagMessage = (command: string) => `
+${chalk.green(`prisma ${command}`)} command is not yet available with ${chalk.green('prisma://')}.
```

Comment:
This was my suggestion from the issue. Not sure if better.
```suggestion
Using the Data Proxy (connection URL starting with protocol ${chalk.green('prisma://')}) is not supported for this CLI command ${chalk.green(`prisma ${command}`)} yet. Please use a direct connection to your database for now.
```
I definitely like how it spells out "Data Proxy" and "connection URL" and "protocol". I don't expect our users to really understand that `prisma://` means = Data Proxy connection string (yet).